pdftitle.py

```diff
@@ -632,12 +632,21 @@ def retrieve_spaces_opt(first_page, title_without_space, p=0, t=0, result=""):
             result += first_page[p]
             t += 1
 
+        # if the current character is the same as the previous
```

Comment:
why is there a need for this ?

Reply:
Suppose I'm looking for the title "Een scheiding" and in the page I find the string "Armoede\nEen scheiding" it will iterate over the page and find "e E" as a temporary result (replacing the newline with a presumed space),  but when it hits the following 'e' it will decide it's not the title we're looking for ("eee" != "een"). This is false, because we can still be on track to find the title, but we should shift the window one character to the right and decide again which is what we're doing here (by not doing t+=1). Maybe I'm overlooking someting, but this solved this use case for me (refer to the trouw.nl PDF I sent separately as a test)